_posts/2019-09-09-what-is-mtls.md

```diff
@@ -15,7 +15,8 @@ class: post-template
 
 Cybersecurity is an ever-growing, ever-complicating field. As a new developer, it can be daunting to think of where to start. Here we're going to talk a little about mTLS and why you as a developer should care about it (and implement it!). 
 
-mTLS is just a type of TLS (Transport Layer Security). This is commonly found in verifying endpoints and for encrypting communications (think of WhatsApp messages) and also browsers (https is a sign of a TLS controlled website). The https part of the url verifies the identity of the server, the webiste you're accessing, to you, the client. Unlike a password which is manually entered by a single user, TLS is typically managed by cryptographic certificates. Most TLS protocols usually verify this with an x509 certificate; and it's typically the client requesting a valid certificate from the server (DNS name on server must match that of the certificate). Secure Sockets Layer (SSL) security is based off this principle (and https is its successor).
+mTLS is just a type of TLS (Transport Layer Security). This is commonly found in verifying endpoints and for encrypting communications (think of WhatsApp messages) and also browsers. The 'https' in a url is a sign of a TLS controlled website. This part of the url verifies the identity of the server, the website you're accessing, to you, the client. 
```

Comment:
Technically speaking, the WhatApp comparison is not true. WhatApp uses [Noise Protocol](http://noiseprotocol.org/).